eq_translations/survey_schema.py

```diff
@@ -86,7 +86,7 @@ def context_pointers(self):
             + self.context_plural_pointers
         )
 
-    def get_core_pointers(self):
+    def _get_core_pointers(self):
```

Comment:
I don't think the methods that return pointers here should be marked protected. The idea here is that any one of these accessor methods are available for scripts to call.

Reply:
I understand but this is wrapped by another method call and is not used elsewhere outside this class currently. But happy to revert if you think it should still be public.

Reply:
I don't think making everything that isn't called outside of the class protected is the way to go. It doesn't make for a nice library design. Less so for the core_pointers, but definitely for the message ones, given that title and list methods haven't changed.

Reply:
This changes in the next commit.